pdbx/containers.py

```diff
@@ -68,11 +69,12 @@ def attribute_part(name):
         if i == -1:
             return None
         else:
-            return name[i + 1:]
+            return name[i + 1 :]  # noqa : E203
```

Comment:
what's the point of having a `noqa` here?

Reply:
It is for flake8 to ignore this particular warning for a space before a colon. The website for black and a few other searches mention that rule E203 is broken. It is supposed to only flag spaces before colons in conditionals (if/while/for) and dictionary terms. This noqa is to tell flake8 that this is for a slice and to ignore it. The pep8 peeve for slices is at https://www.python.org/dev/peps/pep-0008/#pet-peeves